.github/workflows/build.yml

```diff
@@ -23,6 +23,8 @@ jobs:
           - 16-stretch-slim
           - 16-buster-slim
           - 16-bullseye-slim
+          - 18-buster-slim
```

Comment:
What do we think about only providing a bullseye build?

Reply:
I'm not opposed to that idea, and it forces folks to be on the latest version of Debian. I can't think of anything that _wouldn't_ work on Bullseye.

Reply:
Let's do it. If we find something that isn't working, we can always add Buster at that time

Reply:
Sounds good!